pkg/users/account-details.js

```diff
@@ -98,16 +98,27 @@ function get_expire(name) {
 
 export function AccountDetails({ accounts, groups, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
+    const [path, setPath] = useState("");
+
+    useEffect(() => {
+        const lastlog_version = cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
```

Comment:
Calling this "_version" is confusing. Perhaps `utmp_test`? Or don't give it a name in the first place.